python/pyarrow/flight.py

```diff
@@ -33,6 +34,7 @@
     FlightEndpoint,
     FlightInfo,
     SchemaResult,
+    FlightServer,
```

Comment:
I don't know why you're doing this. Now the user gets the impression that there are two distinct server classes, which is false.

Reply:
See https://github.com/apache/arrow/pull/5374#discussion_r323956986